tests/suites/test_suite_ssl.function

```diff
@@ -3487,17 +3575,20 @@ void ssl_decrypt_non_etm_cbc( int cipher_type, int hash_id, int trunc_hmac,
     unsigned char add_data[13];
     unsigned char mac[MBEDTLS_MD_MAX_SIZE];
     int exp_ret;
+    int ret;
     const unsigned char pad_max_len = 255; /* Per the standard */
 
+    USE_PSA_INIT( );
+
     mbedtls_ssl_init( &ssl );
     mbedtls_ssl_transform_init( &t0 );
     mbedtls_ssl_transform_init( &t1 );
 
     /* Set up transforms with dummy keys */
-    TEST_ASSERT( build_transforms( &t0, &t1, cipher_type, hash_id,
+    ret = build_transforms( &t0, &t1, cipher_type, hash_id,
                                    0, trunc_hmac,
                                    MBEDTLS_SSL_MINOR_VERSION_3,
-                                   0 , 0 ) == 0 );
+                                   0 , 0 );
```

Comment:
I think you meant to add `TEST_ASSERT( ret == 0 );` here. The CI picked up on that: `suites/test_suite_ssl.function:3578:9: error: variable 'ret' set but not used [-Werror=unused-but-set-variable]` in `tests/scripts/all.sh build_psa_accel_alg_ecdh` (and other components).

(Btw, I'm not opposed to this change, but I'm not 100% sure why you're doing it. I assume it's because it makes it easier to examine the value of `ret` when debugging?)

Reply:
This seems to be the cause of most errors in the CI, so I stopped checking the CI results after seeing a few occurrences of this one, at this point it will be more efficient to fix this first, and look if any other error remain after that.

Reply:
Yes, your assumption is correct. It was easier to trace errors while debugging. 

I can revert this if needed, but I saw that assignment to `ret` and `TEST_ASSERT( ret == 0 );` is commonly used in the test code,

Reply:
Ok. No need to revert. The present form is more convenient for the next person who'll have to debug something with those tests :) Just wanted to make sure I didn't miss something.